src/pages/CommitDetailPage/CommitCoverage/YamlModal/YamlModalErrorBanner.jsx

```diff
@@ -1,21 +0,0 @@
-import A from 'ui/A'
```

Comment:
I consolidated `YamlModalErrrorBanner` and `YamlErrorBanner` into one and converted from js-->ts